docs/hub/datasets-argilla.md

```diff
@@ -0,0 +1,110 @@
+# Argilla
```

Comment:
You might also want to update https://github.com/huggingface/hub-docs/blob/main/docs/hub/_toctree.yml